docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst

```diff
@@ -131,7 +131,7 @@ We will add the following code to the file.
   - name: Test for new_option, create new user WITH the attribute
     community.postgresql.postgresql_user:
       name: test_user
-      add_attribute: true
```

Comment:
Do these changes in a separate PR.

Reply:
@Akasurde then there needs to be a separate issue to be raised for this in order for me create a PR ?

Reply:
@Akasurde  done in separate PR #79019